app/angular/src/client/index.ts

```diff
@@ -9,6 +9,8 @@ export {
   raw,
 } from './preview';
 
+export { RenderNgAppService } from './preview/angular-beta/RenderNgAppService';
```

Comment:
I hadn't dared to export this service in the public index. ^^

Reply:
I'm not sure this is the right way. However, we recently introduced ESM into the codebase and it broke things: #13013. If there's a less aggressive way to do it, I'm also open to that!